dashboard/package.json

```
@@ -1,8 +1,11 @@
 {
   "private": true,
   "scripts": {
-    "build": "cp -r views dashboard/src",
-    "start": "cd dashboard && npm start",
-    "test": "cd dashboard && npm test"
+    "pull-submodule": "git submodule init && git submodule update",
+    "setup": "npm run pull-submodule && cp -r views dashboard/src && cd dashboard && npm install && cd ..",
```

Would it be possible to use symbolic links instead of copy back and forth the files? This would avoid problems of out of sync changes. An example could be:

```suggestion
    "setup": "npm run pull-submodule && ln -sf $(pwd)/views/crownLabs dashboard/src/views/crownLabs && cd dashboard && npm install && cd ..",
```

---

Sadly at the moment I didn't manage to make it work. Webpack doesn't like very much, although I could not spend much time on it, so it could be worth digging into